src/templates/vm-template-yaml.ts

```diff
@@ -0,0 +1,74 @@
+import { TemplateModel } from '@kubevirt-ui/kubevirt-api/console';
+
+export const defaultVMTemplateYamlTemplate = `
+apiVersion: ${TemplateModel.apiGroup}/${TemplateModel.apiVersion}
+kind: ${TemplateModel.kind}
+metadata:
+  name: vm-template-example
+  labels:
+    template.kubevirt.io/type: vm
+    os.template.kubevirt.io/fedora31: 'true'
```

Comment:
```suggestion
    os.template.kubevirt.io/fedora36: 'true'
```

change to 36 in all places